docs/content/community/media-coverage.md

```diff
@@ -0,0 +1,35 @@
+---
+type: docs
+title: "Radius press coverage and media links"
+linkTitle: "Press/media coverage"
+description: "Learn about Radius through press articles, blog posts, presentation and other coverage of Radius"
+weight: 400
+---
+
+## Blog posts
+
+| Blog Title | Source | Date |
+| ---------- | ------ | ---- |
+|[Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)|Microsoft Azure Blog|Oct 18,2023|
+|[Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)|Microsoft Open Source Blog|Oct 18,2023|
+
+Ongoing blog posts can be found at [Radius Blog](https://blog.radapp.io)
+
+## Videos
+
+| Video | Event | Date |
```

Comment:
For the dates can we consistently have the same sorting? My preference would be newest on top with oldest on bottom.

Reply:
yes updated this in the last commit